operator/internal/central/extensions/reconcile_pvc.go

```diff
@@ -59,32 +76,73 @@ func convertDBPersistenceToPersistence(p *platform.DBPersistence) *platform.Pers
 	if pvc == nil {
 		return &platform.Persistence{}
 	}
+
+	claimName := pvc.ClaimName
+	pvcSize := pvc.Size
+
+	if target == PVCTargetCentralDBBackup {
+		if pvc.ClaimName != nil {
+			// If a ClaimName is specified, derive the backup PVC ClamName from
+			// it as well. We don't want to modify the pointer in place, make a
+			// copy instead -- otherwise the next reconciliation will repeat the
+			// modification, duplicating the suffix.
+			backupName := fmt.Sprintf("%s-backup", *claimName)
+			claimName = &backupName
+		}
+
+		if pvc.Size != nil {
+			// If a Size is specified, derive the backup PVC Size from it as
+			// well, the rule of thumb is that it should be twice as large, to
+			// accomodate the backup and one restore copy.
+			//
+			// The same as above, we don't want to modify the pointer in place,
+			// make a copy instead -- otherwise the next reconciliation will
+			// repeat the modification, duplicating the suffix.
+			quantity, err := resource.ParseQuantity(*pvcSize)
+
+			if err != nil {
+				log.Error(err, "failed to calculate backup volume size")
```

Comment:
I think this (unlikely) error needs to be propagated up and cause a reconcile failure rather than be ignored.

Reply:
I was working on this with the idea that the backup volume logic should not be disruptive for the rest of reconciliation. Bubbling the error up will make it more disruptive instead, are you folks ok with that?

Reply:
I think in this case it's better to shout early than try using a volume that's too small and thus fail in the init container when it runs out of space.